migrations/helpers/update_filtered_positives_helper.py

```diff
@@ -0,0 +1,270 @@
+    """
+    with create_mongo_client(config) as client:
+        mongo_db = get_mongo_db(config, client)
+
+        # get ids of those that are filtered positive, and those that aren't
+        all_ids = [sample[FIELD_MONGODB_ID] for sample in samples]
+        filtered_positive_ids = [
+            sample[FIELD_MONGODB_ID]
+            for sample in list(filter(lambda x: x[FIELD_FILTERED_POSITIVE] is True, samples))
+        ]
+        filtered_negative_ids = [
+            mongo_id for mongo_id in all_ids if mongo_id not in filtered_positive_ids
+        ]
+
+        samples_collection = get_mongo_collection(mongo_db, COLLECTION_SAMPLES)
+        samples_collection.update_many(
+            {FIELD_MONGODB_ID: {"$in": filtered_positive_ids}},
+            {
+                "$set": {
+                    FIELD_FILTERED_POSITIVE: True,
+                    FIELD_FILTERED_POSITIVE_VERSION: version,
+                    FIELD_FILTERED_POSITIVE_TIMESTAMP: update_timestamp,
+                }
+            },
+        )
+        samples_collection.update_many(
+            {FIELD_MONGODB_ID: {"$in": filtered_negative_ids}},
+            {
+                "$set": {
+                    FIELD_FILTERED_POSITIVE: False,
+                    FIELD_FILTERED_POSITIVE_VERSION: version,
+                    FIELD_FILTERED_POSITIVE_TIMESTAMP: update_timestamp,
+                }
+            },
+        )
+    return True
+
+
+def update_mlwh_filtered_positive_fields(config: ModuleType, samples: List[Dict[str, Any]]) -> bool:
+    """Bulk updates sample filtered positive fields in the MLWH database
+
+    Arguments:
+        config {ModuleType} -- application config specifying database details
+        samples {List[Dict[str, str]]} -- the list of samples whose filtered positive fields
+        should be updated
+
+    Returns:
+        bool -- whether the updates completed successfully
+    """
+    mysql_conn = create_mysql_connection(config, False)
+
+    if mysql_conn is not None and mysql_conn.is_connected():
+        mlwh_samples = [map_mongo_to_sql_common(sample) for sample in samples]
+        run_mysql_executemany_query(
+            mysql_conn, SQL_MLWH_MULTIPLE_FILTERED_POSITIVE_UPDATE, mlwh_samples
+        )
+        return True
+    else:
+        return False
+
+
+def update_dart_filtered_positive_fields(config: ModuleType, samples: List[Dict[str, Any]]) -> bool:
+    """Updates DART plates and wells following updates to the filtered positive fields
+
+    Arguments:
+        config {ModuleType} -- application config specifying database details
+        samples {List[Dict[str, str]]} -- the list of samples to update in DART
+
+    Returns:
+        bool -- whether the updates completed successfully
+    """
+    sql_server_connection = create_dart_sql_server_conn(config, False)
+    if sql_server_connection is None:
+        raise ValueError("Unable to establish DART SQL Server connection")
+
+    dart_updated_successfully = True
```

Comment:
I'm tempted to have the default `False`...

Reply:
If the default is `False`, I can't do the `&=` logic explained in the above comment